database-mongo/index.js

```diff
@@ -10,59 +9,75 @@ db.on('error', function() {
 });
 
 db.once('open', function() {
-  //seedDatabase();
+  // seedDatabase();
+  // saveOverviewMain(seed,() => {
+  //   console.log('Data entered');
+  // })
   console.log('mongoose connected successfully');
 });
 
-var overviewSchema = mongoose.Schema({
-  prod_name: String,
-  specs: Array,
-  included: Array,
-  boxContent: Array,
-  descriptionHeader: Array,
-  description: Array,
-  imageHeader: Array,
-  images: Array,
-  shippingDate: String,
-  details: Array
-});
+// var overviewSchema = mongoose.Schema({
+//   prod_name: String,
+//   specs: Array,
+//   included: Array,
+//   boxContent: Array,
+//   descriptionHeader: Array,
+//   description: Array,
+//   imageHeader: Array,
+//   images: Array,
+//   shippingDate: String,
+//   details: Array,
+//   html: String
+// });
+
+// var overviewMainSchema = mongoose.Schema({
+//   html: String
+// })
+
+// var Overview = mongoose.model('Overview', overviewSchema);
+// var overviewMain = mongoose.model('OverviewMain', overviewMainSchema);
 
-var Overview = mongoose.model('Overview', overviewSchema);
+// var saveOverviewMain = (info, callback) => {
+//   console.log(typeof info.file);
+//   let main = new overviewMain({
+//     html: info.file
+//   }).save();
+//   callback(null);
+// };
 
   
-var saveDescription = (info, callback) => {
-  console.log('info looks like : ', info);
-  let overview = new Overview({
-    prod_name: info.prod_name,
-    specs: info.specs,
-    included: info.included,
-    boxContent: info.boxContent,
-    descriptionHeader: info.descriptionHeader,
-    description: info.description,
-    imageHeader: info.imageHeader,
-    images: info.images,
-    shippingDate: info.shippingDate,
-    details: info.details
-  }).save();
-  callback(null);
-};
+// var saveDescription = (info, callback) => {
+//   let overview = new Overview({
+//     prod_name: info.prod_name,
+//     specs: info.specs,
+//     included: info.included,
+//     boxContent: info.boxContent,
+//     descriptionHeader: info.descriptionHeader,
+//     description: info.description,
+//     imageHeader: info.imageHeader,
+//     images: info.images,
+//     shippingDate: info.shippingDate,
+//     details: info.details,
+//     html: info.html
+//   }).save();
+//   callback(null);
+// };
 
-var seedDatabase = () => {
-  console.log(data.file);
-  data.file.forEach((prod) => {
-    console.log(prod)
-    saveDescription(prod, () => {
-      console.log('Inserted');
-    })
-  })
-}
+// var seedDatabase = () => {
+//   data.file.forEach((prod) => {
+//     saveDescription(prod, () => {
+//       console.log('Inserted');
+//     })
+//   })
+// }
```

Comment:
lots of commented out code here!  let's get the schema and model back here and export them for the seed file